README.md

```diff
@@ -198,16 +198,20 @@ Other Style Guides
     // bad
     const atom = {
       value: 1,
-
       addValue: function (value) {
         return atom.value + value;
       },
     };
 
-    // good
+    // very good
```

Comment:
this is actually worse than the following example, which uses concise method syntax.

Reply:
that is a good coding convention.